<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GPRS-485</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>串口连接</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_serial">
           <property name="text">
            <string>串口</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_com"/>
         </item>
         <item>
          <widget class="QLabel" name="label_baud">
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>波特率</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_buad"/>
         </item>
         <item>
          <widget class="QLabel" name="label_byte">
           <property name="text">
            <string>数据位</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_byte"/>
         </item>
         <item>
          <widget class="QLabel" name="label_check">
           <property name="text">
            <string>校验位</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_check"/>
         </item>
         <item>
          <widget class="QLabel" name="label_stop">
           <property name="text">
            <string>停止位</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_stop"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_open_serial">
           <property name="text">
            <string>打开串口</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>参数配置</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_connet">
           <property name="text">
            <string>连接类型</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_connet"/>
         </item>
         <item>
          <widget class="QLabel" name="label_IP_port">
           <property name="text">
            <string>地址与端口</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_ip"/>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_port">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>波特率</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_baudw">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_config_serial">
           <property name="text">
            <string>参数配置</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>查询</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_version">
           <property name="text">
            <string>查询版本</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_signal">
           <property name="text">
            <string>查信号强度</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_in_config">
           <property name="text">
            <string>进入配置</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="pushButton_pin">
           <property name="text">
            <string>查询PIN</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_creg">
           <property name="text">
            <string>注册状态</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_in_check">
           <property name="text">
            <string>进入查询</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="pushButton_greg">
           <property name="text">
            <string>GPRS状态</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_operator">
           <property name="text">
            <string>查运行商</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_out_config">
           <property name="text">
            <string>退出配置</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="pushButton_clear">
        <property name="text">
         <string>清空窗口</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_hex">
        <property name="text">
         <string>hex</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_ascii">
        <property name="text">
         <string>ASCII</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_send"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_send">
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>688</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
